Packs/HelloWorld/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "HelloWorld",
     "description": "This is the Hello World integration for getting started.",
     "support": "community",
-    "currentVersion": "1.2.1",
+    "currentVersion": "1.2.2",
```

i think we can bump a major version
```suggestion
    "currentVersion": "2.0.0",
```